### ISSUE TITLE
Support systemd-networkd?

### ISSUE THREAD

**Issue by ypid** [debops/ansible-ifupdown#41]
I might look into this. networkd is not yet default in Jessie.

**Comment by drybjed** [comment]
This could be interesting, but it probably warrants its own role, instead of putting everything in `debops.ifupdown`.

Actually I'm thinking if network management should be used by default in `common`, or should be done as a separate playbook that's enabled on selected hosts... But since servers are our main use case, setting up bridges by default seems OK to me.

But with separate networkd role networking could be made optional to let users choose what model they want.

**Comment by ypid** [comment]
Right. A separate role is probably better. I first thought about integrating it in `debops.ifupdown` might be a good idea but that would probably blow it up to much. Note sure when I will find time for this.
